Bindings/bindings.json

```diff
@@ -6,473 +6,488 @@
         "appSettingsHelp": "You can use %<App Setting Name>% to reference an App Setting you've already configured",
         "triggerRefHelp": "You can use brackets ({value}) and percent symbols (%value%) to format your path based on parameters from the trigger object."
     },
-    "bindings": [
-        {
-            "type": "timerTrigger",
-            "displayName": "Timer",
-            "direction": "trigger",
-            "defaultParameterName": "myTimer",
-            "parameterNamePrompt":"Name the timestamp object for use in your code",
-            "settings": [
-                {
-                    "name": "schedule",
-                    "value": "string",
-                    "defaultValue": "0 * * * * *",
-                    "required": true,
-                    "label": "How frequently should this function run? Enter a cron expression.",
-                    "help": "A cron expression is a way of expressing a schedule. Please consult the documentation to learn more."
-                },
-                {
-                    "name": "runOnStartup",
-                    "value": "boolean",
-                    "defaultValue": false,
-                    "label": "Should this function run on start up?",
-                    "help": "When true, your function will run on startup. This may be useful for debugging."
-                }
-            ]
+  "bindings": [
```

Comment:
What have we decided on spacing normalization? My local settings are also different that what is checked into this repo